sunpy/map/maputils.py

```diff
@@ -0,0 +1,7 @@
+"""
+Holds utility map function which do not belong in map objects
+"""
+
+
+__all__ = ['test_function']
```

Comment:
Will have to remove this until we add more functions.

Reply:
I think it can just be an empty list.